src/components/Common/ExternalLink.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
```

Comment:
Maybe move it to components/ExternalLink/ExternalLink.tsx?

I think the 'Common' folder is a bit redundant and it is better to try to keep the components flat, unless it is a case where there is a very complex component with internal child components.